parachain/pallets/outbound-queue/src/lib.rs

```diff
@@ -429,13 +429,6 @@ pub mod pallet {
 			fee_per_gas.saturating_mul(max_gas_required.into()).saturating_add(reward)
 		}
 
-		/// Calculate fee in native currency for processing a message locally
-		pub(crate) fn calculate_local_fee() -> T::Balance {
```

Comment:
its better to implement this the other way around. Keep `calculate_local_fee` here, and then just call it in the trait implementation.

That way, all the fee calculation code is kept together

Reply:
Resolved in https://github.com/Snowfork/snowbridge/pull/999/commits/874ecafe216b74bc730950ad433d1a0e732fc9d4